src/templates/components/staffNewsProfile/index.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from 'test-utils'
+import { StaffNewsProfile } from '@/templates/components/staffNewsProfile/index'
+
+describe('StaffNewsProfile', () => {
+  test('renders byline with title and description', () => {
+    const props = {
+      title: 'John Smith',
+      description: 'Staff Writer',
+    }
+    render(<StaffNewsProfile {...props} />)
+    expect(screen.getByText('By John Smith, Staff Writer')).toBeInTheDocument()
+    expect(screen.getByText('By John Smith, Staff Writer').tagName).toBe('P')
```

Comment:
🫠

Reply:
For what it's worth, I just rediscovered the `{ selector: 'p' }` option for these queries. So this could be simply `expect(screen.getByText('By John Smith, Staff Writer', { selector: 'p' })).toBeInTheDocument()`.